#  Write a python program to add all the odd numbers from 0 to 20

In [1]:
odd_num = []
for x in range(1,20,2):
    odd_num.append(x)
print("Odd numbers from 1 to 20 is  :", odd_num)
print("Sum of odd numbers from 1 to 20 is : ", sum(odd_num))
    

Odd numbers from 1 to 20 is  : [1, 3, 5, 7, 9, 11, 13, 15, 17, 19]
Sum of odd numbers from 1 to 20 is :  100


# Write a python program to find the sum of all integers greater than 100 and less than 200.

In [2]:
int_num = []
for x in range(101,200,1):
    int_num.append(x)
print("integres numbers greater than 100 and less than 200 is  :", int_num)
print("Sum of integres numbers greater than 100 and less than 200 is : ", sum(int_num))

integres numbers greater than 100 and less than 200 is  : [101, 102, 103, 104, 105, 106, 107, 108, 109, 110, 111, 112, 113, 114, 115, 116, 117, 118, 119, 120, 121, 122, 123, 124, 125, 126, 127, 128, 129, 130, 131, 132, 133, 134, 135, 136, 137, 138, 139, 140, 141, 142, 143, 144, 145, 146, 147, 148, 149, 150, 151, 152, 153, 154, 155, 156, 157, 158, 159, 160, 161, 162, 163, 164, 165, 166, 167, 168, 169, 170, 171, 172, 173, 174, 175, 176, 177, 178, 179, 180, 181, 182, 183, 184, 185, 186, 187, 188, 189, 190, 191, 192, 193, 194, 195, 196, 197, 198, 199]
Sum of integres numbers greater than 100 and less than 200 is :  14850


#  Write a program to display the sum of square of the first ten even natural numbers

In [3]:
even_natural_num = []
for x in range(2,20,2):
    even_natural_num.append(x)
print("even natural numbers from 1 to 20 is  :", even_natural_num)
print("Sum of even natural numbers from 1 to 20 is : ", sum(even_natural_num))

even natural numbers from 1 to 20 is  : [2, 4, 6, 8, 10, 12, 14, 16, 18]
Sum of even natural numbers from 1 to 20 is :  90


# Write a python program to display ascii characters from 65 to 90

In [4]:
num = range(65,91,1)
for i in num:
    print(" Ascii character of ",i," is : ", chr(i))

 Ascii character of  65  is :  A
 Ascii character of  66  is :  B
 Ascii character of  67  is :  C
 Ascii character of  68  is :  D
 Ascii character of  69  is :  E
 Ascii character of  70  is :  F
 Ascii character of  71  is :  G
 Ascii character of  72  is :  H
 Ascii character of  73  is :  I
 Ascii character of  74  is :  J
 Ascii character of  75  is :  K
 Ascii character of  76  is :  L
 Ascii character of  77  is :  M
 Ascii character of  78  is :  N
 Ascii character of  79  is :  O
 Ascii character of  80  is :  P
 Ascii character of  81  is :  Q
 Ascii character of  82  is :  R
 Ascii character of  83  is :  S
 Ascii character of  84  is :  T
 Ascii character of  85  is :  U
 Ascii character of  86  is :  V
 Ascii character of  87  is :  W
 Ascii character of  88  is :  X
 Ascii character of  89  is :  Y
 Ascii character of  90  is :  Z


# Display ascii characters from 48 to 57.

In [5]:
num = range(48,58,1)
for i in num:
    print(" Ascii character of ",i," is : ", chr(i))

 Ascii character of  48  is :  0
 Ascii character of  49  is :  1
 Ascii character of  50  is :  2
 Ascii character of  51  is :  3
 Ascii character of  52  is :  4
 Ascii character of  53  is :  5
 Ascii character of  54  is :  6
 Ascii character of  55  is :  7
 Ascii character of  56  is :  8
 Ascii character of  57  is :  9


# Display the 97 to 122 output with the help of Ascii character.

In [6]:
num = range(97,123,1)
for i in num:
    print(" Ascii character of ",i," is : ", chr(i))

 Ascii character of  97  is :  a
 Ascii character of  98  is :  b
 Ascii character of  99  is :  c
 Ascii character of  100  is :  d
 Ascii character of  101  is :  e
 Ascii character of  102  is :  f
 Ascii character of  103  is :  g
 Ascii character of  104  is :  h
 Ascii character of  105  is :  i
 Ascii character of  106  is :  j
 Ascii character of  107  is :  k
 Ascii character of  108  is :  l
 Ascii character of  109  is :  m
 Ascii character of  110  is :  n
 Ascii character of  111  is :  o
 Ascii character of  112  is :  p
 Ascii character of  113  is :  q
 Ascii character of  114  is :  r
 Ascii character of  115  is :  s
 Ascii character of  116  is :  t
 Ascii character of  117  is :  u
 Ascii character of  118  is :  v
 Ascii character of  119  is :  w
 Ascii character of  120  is :  x
 Ascii character of  121  is :  y
 Ascii character of  122  is :  z


# Write a python program for given a Python list you should be able to display Python list in the following order :
    L1 = [100, 200, 300, 400, 500]
    Expected output : [500, 400, 300, 200, 100]

In [7]:
Given_list = [100, 200, 300, 400, 500]
Given_list.reverse()
print("Expected output : ", Given_list)

Expected output :  [500, 400, 300, 200, 100]


# Write a Python program to concatenate following dictionaries to create a new one.
    Sample Dictionary : 
    dic1={1:10, 2:20} 
    dic2={3:30, 4:40} 
    dic3={5:50,6:60}
    Expected Result : {1: 10, 2: 20, 3: 30, 4: 40, 5: 50, 6: 60}

In [8]:
dic1={1:10, 2:20} 
dic2={3:30, 4:40} 
dic3={5:50,6:60}

In [9]:
expected_Result = {**dic1, **dic2, **dic3}
expected_Result

{1: 10, 2: 20, 3: 30, 4: 40, 5: 50, 6: 60}

# Write a Python program to add key to a dictionary.
    Sample Dictionary : {0: 10, 1: 20}
    Expected Result : {0: 10, 1: 20, 2: 30}

In [10]:
Sample_Dictionary = {0: 10, 1: 20}
Sample_Dictionary[2] = 30
print("Expected_Result :", Sample_Dictionary)

Expected_Result : {0: 10, 1: 20, 2: 30}


# Write a Python program to print out a set containing all the colors from a list which are not present in another list
    Test Data : 
    color_list_1 = set(["White", "Black", "Red"]) 
    color_list_2 = set(["Red", "Green"])

In [11]:
colour_list_1 = set(["White", "Black", "Red"]) 
colour_list_2 = set(["Red", "Green"])

In [12]:
print("Expected Output :",colour_list_1 - colour_list_2)

Expected Output : {'White', 'Black'}


# Given a Python list. Write a python program to turn every item of a list into its square 
    List1 = [1, 2, 3, 4, 5, 6, 7]
    Expected output:[1, 4, 9, 16, 25, 36, 49]

In [13]:
List1 = [1, 2, 3, 4, 5, 6, 7]
output = []
for i in List1:
    output.append(i**2)
print("Expected output :", output)
    

Expected output : [1, 4, 9, 16, 25, 36, 49]


# Program to count the number of each vowel in a string.

In [14]:
string = str(input("Enter string : "))
vowel = ['a','e','i','o','u']
for i in vowel:
    count = string.count(i)
    print("count of vowel",i," in enter string is :",count)


Enter string : python is amazing language
count of vowel a  in enter string is : 4
count of vowel e  in enter string is : 1
count of vowel i  in enter string is : 2
count of vowel o  in enter string is : 1
count of vowel u  in enter string is : 1


# Write a python program to Access the value of key ‘history’ from the following dictionary 
sampleDict = {
"class":{
"student":{
"name":"Mike",
"marks":{
"physics":70,
"history":80
}
}
}
}


In [15]:
sampleDict = { "class":{ "student":{ "name":"Mike", "marks":{ "physics":70, "history":80 } } } }
Expected_output = print("Value of key 'history' is : ",sampleDict['class']['student']['marks']['history'])

Value of key 'history' is :  80


# write a Program to Remove Punctuations from a String provided by the user. [Hint: use punctuation attribute of string module to get all punctuations 
    (i.e. !"#$%&\'()*+,-./:;<=>?@[\\]^_`{|}~ ) 

In [16]:
import string
sample_string = str(input(" Enter the string with punctuaton : "))

 Enter the string with punctuaton : #python^ is@ %amazing? and {simple}


In [17]:
Expected_output = sample_string.translate(str.maketrans('','',string.punctuation))
print("Expected Output is  : ", Expected_output)

Expected Output is  :  python is amazing and simple


# Write a python program to print the Following:
    1
    2 1
    3 2 1

In [18]:
for i in range(1,2):
    for j in range(2,3):
        for k in range(3,4):
            print(i)
            print(j,' ',i)
            print(k,' ',j,' ',i)

1
2   1
3   2   1


# WAP to print the following asterisk pattern:
    *
    * *
    * * * 
    * * * *
    * * * * *
    * * * * * *

In [19]:
pattern = str('*')
print ('Expected Output :')
for i in range(1,7):
    print('* ' * i)

Expected Output :
* 
* * 
* * * 
* * * * 
* * * * * 
* * * * * * 


# WAP to create a function traiangle to print the following asterisk triangle pattern:
    *
    * *
    * * *
    * * * *
    * * *
    * *
    *

In [20]:
pattern = str('*')
print ('Expected Output :')
for i in range(1,5):
    print('* ' * i)
for i in range(3,0,-1):
    print('* ' * i)

Expected Output :
* 
* * 
* * * 
* * * * 
* * * 
* * 
* 


# Write a python program to print multiplication table from 1 to 10

In [21]:
for i in range(1,11):
    print('  ',i,end='\t')
print ('')
print ('')
print('-' * 79)
for j in range(1,11):
    for k in range(1,11):
        print('  ', j*k,end='\t')
    print('\n')

   1	   2	   3	   4	   5	   6	   7	   8	   9	   10	

-------------------------------------------------------------------------------
   1	   2	   3	   4	   5	   6	   7	   8	   9	   10	

   2	   4	   6	   8	   10	   12	   14	   16	   18	   20	

   3	   6	   9	   12	   15	   18	   21	   24	   27	   30	

   4	   8	   12	   16	   20	   24	   28	   32	   36	   40	

   5	   10	   15	   20	   25	   30	   35	   40	   45	   50	

   6	   12	   18	   24	   30	   36	   42	   48	   54	   60	

   7	   14	   21	   28	   35	   42	   49	   56	   63	   70	

   8	   16	   24	   32	   40	   48	   56	   64	   72	   80	

   9	   18	   27	   36	   45	   54	   63	   72	   81	   90	

   10	   20	   30	   40	   50	   60	   70	   80	   90	   100	

